README.md

```diff
@@ -45,6 +45,11 @@ At terminal window, click `Edit` > `Profile Preferences`, click on `Title and Co
 
 ![Terminal Screenshot](terminal.png)
 
+Or instead, you can run the following command 
+```
+echo 'source "/etc/profile.d/rvm.sh"' >> ~/.bashrc
+```
+
```

Comment:
what's important is that RVM is very interactive, RVM might and will ask questions on screen, this will interfere with your system in bizarre ways

Reply:
@mpapis Should I revert the merge?